redirect/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     testImplementation "junit:junit:$junit_version"
     androidTestImplementation "androidx.test.ext:junit:$test_ext_version"
     androidTestImplementation "androidx.test.espresso:espresso-core:$espresso_version"
+    testImplementation "org.robolectric:robolectric:$robolectric_version"
```

Comment:
Is it only me that gets triggered by the order here? 😛

Reply:
You're right I didn't notice it